spring-boot/current-community/istio-security/booster.yaml

```diff
@@ -2,10 +2,10 @@ metadata:
   app:
     launcher:
       runsOn:
-        - 'local'
+      - local
```

Comment:
This looks like it's not indented correctly, can you fix it?

Reply:
I'm actually trying to automate the launcher update. This PR was the result of an attempt at such. I still have several issues to fix but I'm not sure I'll be able to avoid the reformats since I use `yq` which converts YAML into JSON and feeds it into `jq` then back to YAML… I would assume that the reformating leaves the semantics intact, though (otherwise the tool would be useless).